README.md

```diff
@@ -88,27 +114,27 @@ The content is defined in YAML files and most fields are optional. If they are n
 
 - field_name_base_url: `definition_table` will generate nested links for each field prepended with this url
 - fields: The list of defined fields
```

Comment:
Do we need a period at the end?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/558/commits/04ff73228f37e8ce929e1a65db7ed509e88040c3#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R116